osu.Game/Screens/Select/FooterButtonRandom.cs

```diff
@@ -67,5 +56,36 @@ private void updateText()
                 secondaryText.FadeOut(120, Easing.InQuad);
             }
         }
+
+        public override bool OnPressed(GlobalAction action)
+        {
+            switch (action)
+            {
+                case GlobalAction.SelectPreviousRandom:
+                    secondaryActive = true;
```

Comment:
probably rename `secondaryActive` to `reverseSearching` or something similar.

Reply:
Renamed to rewindSearch, is that fine?

Reply:
i guess that will do.